src/components/course/data/utils.jsx

```diff
@@ -932,11 +971,30 @@ export function getSoonestEarliestPossibleExpirationData({
   if (dateFormat) {
     soonestExpirationDate = dayjs(soonestExpirationDate).format(dateFormat);
   }
-
   return {
     soonestExpirationDate,
     soonestExpirationReason: sortedByExpirationDate[0].earliestPossibleExpiration.reason,
     soonestExpiringAssignment: sortedByExpirationDate[0],
     sortedExpirationAssignments: sortedByExpirationDate,
   };
 }
+
+/**
+ * If the start date of the course is before today offset by the START_DATE_DEFAULT_TO_TODAY_THRESHOLD_DAYS
+ * then return today's formatted date. Otherwise, pass-through the start date in ISO format.
+ *
+ * For cases where a start date does not exist, just return today's date.
+ *
+ * @param start
+ * @param format
+ * @returns {string}
+ */
+export const setStaleCourseStartDates = ({ start }) => {
```

Comment:
nit: Similar to Admin Portal feedback (IIRC), the name of this function with the `set` prefix seems to suggest its a React state setter function, which isn't the case.

Reply:
Actually, related, I think my question here is actually whether this function overlaps with the `getNormalizedStartDate` above?

Reply:
Removed, unused duplicate code.